test/DGmethods/compressible_Navier_Stokes/rising_bubble-model.jl

```diff
@@ -0,0 +1,202 @@
+# Load Packages 
+using MPI
+using CLIMA
+using CLIMA.Mesh.Topologies
+using CLIMA.Mesh.Grids
+using CLIMA.Mesh.Geometry
+using CLIMA.DGmethods
+using CLIMA.DGmethods.NumericalFluxes
+using CLIMA.MPIStateArrays
+using CLIMA.LowStorageRungeKuttaMethod
+using CLIMA.SubgridScaleParameters
+using CLIMA.ODESolvers
+using CLIMA.GenericCallbacks
+using CLIMA.Atmos
+using CLIMA.VariableTemplates
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using LinearAlgebra
+using StaticArrays
+using Logging, Printf, Dates
+using CLIMA.VTK
+using Random
+
+@static if haspkg("CuArrays")
+  using CUDAdrv
+  using CUDAnative
+  using CuArrays
+  CuArrays.allowscalar(false)
+  const ArrayTypes = (CuArray,)
+else
+  const ArrayTypes = (Array,)
+end
+
+if !@isdefined integration_testing
+  const integration_testing =
+    parse(Bool, lowercase(get(ENV,"JULIA_CLIMA_INTEGRATION_TESTING","false")))
+end
+
+# -------------- Problem constants ------------------- # 
+const xmin      = 0
+const ymin      = 0
+const zmin      = 0
+const xmax      = 1000
+const ymax      = 400
+const zmax      = 1000
+const Ne        = (10,2,10)
+const polynomialorder = 4
+const dim       = 3
+const dt        = 0.01
+const timeend   = 10dt
+# ------------- Initial condition function ----------- # 
+function initialise_rising_bubble!(state::Vars, aux::Vars, (x1,x2,x3), t)
+  DF            = eltype(state)
+  R_gas::DF     = R_d
```

Comment:
I think it is repetitive to define new constants that are simply equal to the global constants.